src/account/accountsReducer.js

```diff
@@ -62,8 +34,7 @@ const accountSwitch = (state, action) => {
 const findAccount = (state: AccountsState, identity: Identity): number => {
   const { realm, email } = identity;
   return state.findIndex(
-    account =>
-      account.realm.toString() === realm.toString() && (!account.email || account.email === email),
+    account => account.realm.toString() === realm.toString() && account.email === email,
```

Comment:
Along with this cleanup, we can delete the `, email` here below:
```js
  return [
    { ...state[accountIndex], email, apiKey, ackedPushToken: null },
```

That's now a no-op because at that point we've already ensured that the account we find at `accountIndex` is one that has a matching `email`. It used to be needed for the case of filling in a partial auth.